src/main/java/funbox/task/Task.java

```diff
@@ -0,0 +1,66 @@
+package funbox.task;
+
+//@@author junrong98-reused
+//Reused from https://nus-cs2103-ay2122s2.github.io/website/admin/ip-w2.html
+// with minor modifications
+
+/**
+ * The task class represents the task user's wants to do
+ */
+public class Task {
+    public String description;
+    public boolean isDone;
+    public String type;
+
+    /**
+     * Constructor for the Task.
+     *
+     * @param description The description of the task.
+     * @param type The type of task (Deadline, Event, Todo).
+     */
+    public Task(String description, String type) {
+        this.description = description;
+        this.isDone = false;
+        this.type = type;
+    }
+
+    /**
+     * The icon representing the current status of the task
+     *
+     * @return Return "X" if status of task is completed else return " "
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Not sure how strict they are, but the coding standards provide these 2 formats as acceptable:

> Here are two acceptable ways to format ternary expressions:

```java
alpha = (aLongBooleanExpression) ? beta : gamma;
alpha = (aLongBooleanExpression)
        ? beta
        : gamma;
```

```suggestion
        return (isDone) ? "X" : " "; // mark done task with X
```